### PR TITLE
Added unit tests for delete debate views and edit roles (spaces) tests.

### PR DESCRIPTION
**Pull request by piyushbansal**

### CODE REVIEW

File: tests/unit_tests/src/core/spaces/test_spaces.py [hunk 1]
```diff
@@ -207,3 +207,101 @@ def testAllSpacesAreReturnedForALoggedInUser(self):
...
+        response=self.get(url)
+        self.assertResponseOK(response)
+        self.assertTemplateUsed(response, 'not_allowed.html')
+
+
+
+
+
+    
+
```
Comment (praveen97uma) [inline tests/unit_tests/src/core/spaces/test_spaces.py]: Please remove all these blank lines at the end of this file.

**Comment by praveen97uma** [comment]
All your methods should be in camel case with the first letter being small. So instead of OtherUserCannotAccessPublicView, it should be otherUserCannotAccessPublicView. Please correct it 
in everywhere.

**Comment by praveen97uma** [comment]
You should write good and informative commit messages. Messages like "minor error", "indentation error" etc doesn't tell me what the commit is for. All the  messages should start with a verb. One way of writing "all methods are camelcased with first letter small" could be "Make all method names camelCased."
"minor error" -> Fix a minor mistake in something.py.
"indentatiion errors" -> Fix indentation errors.
All the commit messages should start with a capital letter and end with a full stop.

**Comment by piyushbansal** [comment]
@praveen97uma : thanks, point noted !

**Pull request merged**